bigint_func.circom

```diff
@@ -0,0 +1,442 @@
+pragma circom 2.0.2;
+
+function isNegative(x) {
+    // half babyjubjub field size
+    return x > 10944121435919637611123202872628637544274182200208017171849102093287904247808 ? 1 : 0;
+}
+
+function div_ceil(m, n) {
+    var ret = 0;
+    if (m % n == 0) {
+        ret = m \ n;
+    } else {
+        ret = m \ n + 1;
+    }
+    return ret;
+}
+
+function log_ceil(n) {
```

Comment:
Doesn't this function return log_ceil(2) = 2 and log_ceil(4) = 3? Shouldn't the ceiling of log(2) be 1 and ceiling of log(4) be 2 since they are exact?

Reply:
If this function is used to calculate how many bits are needed to represent "n", then it makes sense. Just need to be careful of log_ceil(0) == 0